### ISSUE TITLE
Visualising multiple Knockout View Models in BladesetWorkbench does not work

### ISSUE THREAD

**Issue by ioanalianabalas** [BladeRunnerJS/brjs#1196]
When adding multiple blades for test in the BladesetWorkbench, the associated Knockout View Model will only be displayed for the first one. For the second blade, the panel will show but it will be empty.

If I try finding the associated view model with the 'find' button in the second blade, it will display the correct information, but it will remove all the content of the other Knockout View Model panel. 

Basically, as it stands, we can only show information in one Knockout View Model at a time.

**Comment by ioanalianabalas** [comment]
This is dependant on #1238 so I will postpone it until that one is fixed.

**Comment by ioanalianabalas** [comment]
This is a duplicate of #1135. This issue will be closed in its favour, and all comments have been reposted there.
